spec/support/acceptance/prepare_host.rb

```diff
@@ -5,21 +5,28 @@ def prepare_host
 
   apply_manifest <<~PUPPET
     $services = $facts['os']['family'] ? {
-      'RedHat' => ['zabbix-server', 'httpd'],
-      'Debian' => ['zabbix-server', 'apache2'],
-      default  => [],
+      'RedHat' => ['zabbix-server', 'httpd', 'zabbix-agent', 'zabbix-agent2'],
+      'Debian' => ['zabbix-server', 'apache2',  'zabbix-agent', 'zabbix-agent2'],
+      default  => ['zabbix-agent', 'zabbix-agent2'],
     }
     service { $services:
       ensure => stopped
     }
 
     $packages = $facts['os']['family'] ? {
-      'RedHat' => ['zabbix-server-pgsql', 'zabbix-server-pgsql-scl', 'zabbix-web', 'zabbix-web-pgsql', 'zabbix-web-pgsql-scl', 'zabbix-frontend-php', 'zabbix-sql-scripts'],
-      'Debian' => ['zabbix-server-pgsql', 'zabbix-web-pgsql', 'zabbix-frontend-php', 'zabbix-sql-scripts'],
-      default  => [],
+      'RedHat' => ['zabbix-server-pgsql', 'zabbix-server-pgsql-scl', 'zabbix-web', 'zabbix-web-pgsql', 'zabbix-web-pgsql-scl', 'zabbix-frontend-php', 'zabbix-sql-scripts', 'zabbix-agent', 'zabbix-agent2'],
+      'Debian' => ['zabbix-server-pgsql', 'zabbix-web-pgsql', 'zabbix-frontend-php', 'zabbix-sql-scripts', 'zabbix-agent', 'zabbix-agent2'],
+      default  => ['zabbix-agent', 'zabbix-agent2'],
     }
+
+    $pkg_ensure = $facts['os']['family'] ? {
+      'RedHat' => purged,
+      'Debian' => purged,
+      default  => absent,
+    }
```

Comment:
Is that block required? Which OS doesn't support `purged`?

Reply:
Archlinux not support purged, I don't check the other, you prefer to use purged as default and set only on archlinux absent?

Reply:
Oh this is indeed a bug within Puppet

Reply:
https://github.com/puppetlabs/puppet/pull/9287

Reply:
I've restore purged as default, it fail on archlinux before the PR was merged and released

Reply:
While waiting for the release I set purged as default and absent for Archlinux